src/main/resources/com/rsmart/kuali/kfs/cr/businessobject/datadictionary/CheckReconciliation.xml

```diff
@@ -275,6 +281,8 @@ http://www.springframework.org/schema/beans http://www.springframework.org/schem
               p:attributeName="status" />
         <bean parent="FieldDefinition"
               p:attributeName="sourceCode" />
+        <bean parent="FieldDefinition"
+              p:attributeName="active" />

```

Comment:
Are these changes part of KFSPTS-30132? I don't think this file should be in this PR.